samples/azure-quantum/qae-numerical-integration/QuantumAmplitudeEstimation.ipynb

```diff
@@ -0,0 +1,1148 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot(ys, k_vals)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this plot we see that all of our results look fairly close to the expected value graphed in green.\n",
+    "\n",
+    "Therefore, we'll graph the errors to get see which $k$ performs best."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def epsilon_squared(x : float, expected : float) -> float:\n",
```

Comment:
Ah, this defines the loss function explicitly, got it. I'd suggest naming it to reflect that, rather than in terms of a mathematical symbol (ε).